### PR TITLE
Feature/close pictures

### PR DESCRIPTION
**Pull request by BoostedBoat**
This features lets a user decide a radius in which the next selected pictures will be chosen

### CODE REVIEW

File: app/src/main/java/com/github/wnder/user/User.java [hunk 1]
```diff
@@ -27,5 +40,32 @@ public Location getPositionFromGPS(LocationManager manager, Context context){
...
+     * @param idsAndLocs the ids associated with the locations of the pictures
+     * @return a set of string with the ids of the pictures respecting the criteria
+     */
+    protected Set<String> keepOnlyInRadius(Map<String, Location> idsAndLocs){
+        Set<String> correctIds = new HashSet<>();
+        for(Map.Entry<String, Location> entry : idsAndLocs.entrySet()){
+            float[] res = new float[1];
+            //TODO: replace with location getter from leonard
+            Location.distanceBetween(entry.getValue().getLatitude(), entry.getValue().getLongitude(), location.getLatitude(), location.getLongitude(), res);
+            Resources resources;
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/user/User.java:64]: Is this line ever used?

File: app/src/main/java/com/github/wnder/Storage.java [hunk 2]
```diff
@@ -100,4 +102,35 @@ public void onFailure(@NonNull Exception e) {
...
+                    idsAndLoc.put(docs.get(i).getId(), loc);
+                }
+                idsAndLocsToRet.complete(idsAndLoc);
+            }
+        }).addOnFailureListener(new OnFailureListener() {
+            @Override
+            public void onFailure(@NonNull Exception e) {
+                idsAndLocsToRet.complete(new HashMap<>());
+            }
+        });
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/Storage.java:133]: Use lambdas

**Review by rkuenzi-epfl — CHANGES_REQUESTED** [review]
Use lambdas!

File: app/src/main/java/com/github/wnder/user/User.java (hunk 1 above)
Comment (BoostedBoat) [reply app/src/main/java/com/github/wnder/user/User.java]: removed and never to be seen again

File: app/src/main/java/com/github/wnder/Storage.java (hunk 2 above)
Comment (BoostedBoat) [reply app/src/main/java/com/github/wnder/Storage.java]: tried to do something

**Review by rkuenzi-epfl — APPROVED** [review]

**Pull request merged**